lib/block_view.go

```diff
@@ -288,6 +363,12 @@ type GlobalParamsEntry struct {
 	// The new create profile fee
 	CreateProfileFeeNanos uint64
 
+	// The fee to create a single NFT (NFTs with n copies incur n of these fees).
+	CreateNFTFeeNanos uint64
+
+	// The maximum number of NFT copies that are allowed to be minted.
+	MaxCopiesPerNFT uint64
```

Comment:
Fun idea @redpartyhat @lazynina. What if instead of setting a block height at which this activates we instead do the following:
* Make sure MaxCopiesPerNFT defaults to zero initially
* We make the new _connect*Txn() functions for NFTs reject automatically until MaxCopiesPerNFT is non-zero. Suppose we add this as an explicit check at the top of all the _connect functions.
* We wait for Blockchain, Ascendex, Coinbase, etc... to upgrade
* Then, once we're sure everyone is upgraded, we update MaxCopiesPerNFT and trigger the product.

This has the upshot of also solving that weird problem where MaxCopiesPerNFT is zero by default on old nodes.